latex/README.md

```
@@ -0,0 +1,29 @@
+
+macOS:
+
+```
+brew install pandoc 
+brew install imagemagick
+brew install ghostscript
+brew install jq
+brew install --cask mactex
+npm install --global mermaid-filter @mermaid-js/mermaid-cli
+brew install inkscape
+```
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)